tests/test_controller.py

```diff
@@ -383,6 +385,48 @@ def test_edit_registration_with_override(self):
             eq_(response.response[0], nypl.internal_urn)
             edited_nypl = get_one(self._db, Library, internal_urn=nypl.internal_urn)
 
+    def test_email(self):
+        nypl = self.nypl
+        uuid = nypl.internal_urn.split("uuid:")[1]
+        with self.app.test_request_context("/", method="POST"):
+            flask.request.form = MultiDict([
+                ("uuid", uuid),
+            ])
+            response = self.controller.email()
+        eq_(response.response, [nypl.internal_urn])
+        eq_(response.status_code, 200)
```

Comment:
This needs to also check that an email was actually sent. You should be able to access self.controller.emailer.sent_out and see the email in there.